model/Dataset/Classes/Dataset.md

```diff
@@ -68,3 +66,32 @@ Metadata information that can be added to a dataset that may be used in a softwa
   - type: DatasetAvailabilityType
   - minCount: 0
   - maxCount: 1
+
+## External properties restrictions
+
+- /Core/Element/name
+  - minCount: 1
+- /Core/Properties/originatedBy
+  - minCount: 1
+- /Core/Properties/suppliedBy
+  - minCount: 0
+- /Core/Hash
+  - minCount: 0
+- /Software/Package/downloadLocation
+  - minCount: 1
+- /Software/Package/packageVersion
+  - minCount: 0
+- /Software/Properties/purpose
+  - minCount: 1
+- /Core/Element/comment
+  - minCount: 0
+- /Licensing/Properties/concludedLicense
+  - minCount: 1
+- /Licensing/Properties/declaredLicense
+  - minCount: 1
+- /Core/Properties/releaseTime
+  - minCount: 1
```

Comment:
So, only released Datasets can be described?

Reply:
As specified now,  we may need to revisit it being mandatory for streaming data.   Best to put in an issue for this so we don't overlook.